swagger/v1/swagger.yaml

```diff
@@ -77,7 +77,7 @@ components:
                 billing_address:
                   type: object
                   nullable: true
-                  example:
+                  example: 
```

Comment:
These changes seem unrelated and don't make any sense. Are newer version of Swagger causing these? It's good to commit unrelated changes separately.

Reply:
I dont know why this happened. I can split it if you want